src/xo-app/settings/remotes/index.js

```diff
@@ -0,0 +1,313 @@
+import _, { messages } from 'messages'
+import ActionButton from 'action-button'
+import filter from 'lodash/filter'
+import Icon from 'icon'
+import isEmpty from 'lodash/isEmpty'
+import map from 'lodash/map'
+import React, { Component } from 'react'
+import { connectStore } from 'utils'
+import { error } from 'notification'
+import { injectIntl } from 'react-intl'
+import {
+  createRemote,
+  deleteRemote,
+  disableRemote,
+  enableRemote,
+  subscribeRemotes
+} from 'xo'
+import {format, parse} from 'xo-remote-parser'
```

Comment:
Try to leave space at the beginning and end of arrays/objects, it might one day be enforced by standard.